AngularApp/projects/app-service-diagnostics/src/app/shared/components/tools/network-checks/network-check-flows/functionsFlow.js

```diff
@@ -190,31 +188,31 @@ export var functionsFlow = {
             if (functionsInfo.length == 0) {
                 return new InfoStepView({ infoType: 0, title: "No functions with configured connection strings were found for this Function App" });
             }
-
             var subChecksL1 = [];
+
             var promisesL1 = functionsInfo.map(async (functionInfo) => {
                 var subChecksL2 = []; // These are the checks (and subchecks) for each binding of a function
                 var promisesL2 = functionInfo.bindings.map(async (binding) => {
                     var connectionString = binding.connectionString;
+                    var failureDetailsMarkdown = undefined;                    
                     // An undefined connectionStringType parameter causes the old tcpping validation to apply
-                    var connectionStringType = isDaasExtAccessible ? bindingTypeToConnectionStringType(binding.type) : undefined;
-                    if (connectionString != undefined && connectionString != "") {
+                    if(isDaasNew){

```

Comment:
Why do you need this if and else when bindingTypeToConnectionStringType() takes the isDasNew flag?  The code can be compacted to a single line instead of the 5 lines here.

Reply:
fixed